### PR TITLE
Make 'Fork Me on GitHub' ribbon

### PR DESCRIPTION
**Pull request by sandylcruz**
* Make "Fork Me on GitHub" ribbon
* Have it disappear for mobile devices

fork-demo
Screen Shot 2021-11-16 at 11 03 21 AM

### CODE REVIEW

File: components/ForkRibbon/ForkRibbon.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const ForkRibbon = ({
+  backgroundColor = "black",
+  repoLink = "https://github.com/gravitational/teleport",
+  text = "Fork Me on GitHub",
+}: ForkRibbonProps) => {
+  return (
+    <OutermostContainer>
+      <OuterContainer>
+        <RibbonContainer backgroundColor={backgroundColor || "black"}>
+          <StyledLink
```
Comment (C-STYR) [inline components/ForkRibbon/ForkRibbon.tsx:25]: Please review the `Link` component and it's functionality, specifically the props included for external hrefs (starting around ln 88)

File: components/ForkRibbon/ForkRibbon.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          >
+            {text}
+          </StyledLink>
+        </RibbonContainer>
+      </OuterContainer>
+    </OutermostContainer>
+  );
+};
+
+const OuterContainer = styled("div")(
```
Comment (C-STYR) [inline components/ForkRibbon/ForkRibbon.tsx:39]: For this and subsequent local components that do not contain functional code (only CSS), please prefix with `Styled`

File: components/ForkRibbon/ForkRibbon.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+const RibbonContainer = styled("div")<RibbonContainerProps>(
+  css({
+    border: "1px solid white",
+    boxShadow: "2px 2px 8px rgba(50, 50, 50, 0.75)",
+    color: "white",
+    cursor: "pointer",
+    height: "32px",
+    marginRight: 4,
+    transform: "translate(32px, -24.041px) rotate(45deg)",
```
Comment (C-STYR) [inline components/ForkRibbon/ForkRibbon.tsx:68]: Nice!

File: components/ForkRibbon/ForkRibbon.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+const ForkRibbon = ({
+  backgroundColor = "black",
+  repoLink = "https://github.com/gravitational/teleport",
+  text = "Fork Me on GitHub",
+}: ForkRibbonProps) => {
+  return (
+    <OutermostContainer>
+      <OuterContainer>
+        <RibbonContainer backgroundColor={backgroundColor || "black"}>
```
Comment (C-STYR) [inline components/ForkRibbon/ForkRibbon.tsx:24]: Good choice of optional params here.

**Review by C-STYR — COMMENTED** [review]
Good job, just a little cleanup and you're there.

**Comment by C-STYR** [comment]
Also, all new components require a Storybook entry please!

File: components/ForkRibbon/ForkRibbon.tsx (hunk 1 above)
Comment (sandylcruz) [reply components/ForkRibbon/ForkRibbon.tsx]: Removed duplicate props

File: components/ForkRibbon/ForkRibbon.tsx (hunk 2 above)
Comment (sandylcruz) [reply components/ForkRibbon/ForkRibbon.tsx]: Fixed

**Review by C-STYR — APPROVED** [review]

**Pull request merged**